src/main/webapp/WEB-INF/jsp/scoreboard.jsp

```diff
@@ -228,6 +228,7 @@
                             <div class="panel-body game-clock">
                                 <div class="digits-alt">
                                     <span id="gameClockMins">${score.gameClock.mins}</span>:<span id="gameClockSecs">${score.gameClock.secs}</span>
+                                    <!--<span id="gameClockMins">05</span>:<span id="gameClockSecs">35</span>-->
```

Comment:
You've added a commented out line with no other changes.  Is this what you meant to do?